integration/entity.go

```diff
@@ -18,6 +18,8 @@ type Entity struct {
 	Events    []*event.Event       `json:"events"`
 	storer    persist.Storer
 	lock      sync.Locker
+	// CustomAttributes []metric.Attribute `json:"custom_attributes,omitempty"`
+	customAttributes []metric.Attribute
```

Comment:
This here is just commented because we could do it either by adding the custom attributes to the samples in the integration. Or add them to the entities and then in our backend decorate all the samples from that entity. That's why I've left this here to show how we could do it.